introduction.tex

```diff
@@ -124,6 +124,9 @@ \subsubsection{Incompatible Changes from 0.13 to 1.0}
     can be written with any value supported by any of the types this trigger
     supports. \PR{721}
     \item \RcsrTcontrol fields only apply to breakpoint traps, not any trap. \PR{723}
+    \item \FcsrMcontrolSixHitZero now contains 0 when a trigger fires more than one instruction
+        after the instruction that matched. (This information is now reflected
+        in \FcsrMcontrolSixHitOne.) \PR{795}
```

Comment:
This should be prefaced by the fact that hit is renamed to FcsrMcontrolSixHitZero.

Reply:
That's not a backwards incompatible change, though. What matters for compatibility is the location of the bit and what values it takes when.

Reply:
Someone here read this and was concerned about the word "now" which implies that there previously was a hit0 field and the behavior of that field is "now" different.  I was trying to address that confusion.

Reply:
Now I understand your point.  It has to do with the way that encoding 2 would be interpreted by old debuggers.  I still think that some minor explanation of hit vs. hit0 is useful to people who may be scratching their head about why the hit0 behavior changed when there was no hit0 in the old spec.

```
    \item The hit bit is now called \FcsrMcontrolSixHitZero and it
        contains 0 when a trigger fires more than one instruction
        after the instruction that matched. (This information is now reflected
        in \FcsrMcontrolSixHitOne.) \PR{795}
```

Reply:
Or
```
    \item \FcsrMcontrolSixHitZero (called "hit" in 0.13) now contains 0 when a trigger fires more than one instruction
        after the instruction that matched. (This information is now reflected
        in \FcsrMcontrolSixHitOne.) \PR{795}
```

Reply:
I added the 2nd version.